src/changes/changes.xml

```diff
@@ -24,5 +24,12 @@
 				Return all borrowed books by one member.
 			</action>
 		</release>
+
+		<release version="1.1" date="2022-20-10"
+			description="Updated techstack">
```

Comment:
description is for the whole version, not a single feature. Hence, we could use the sprint goal for example as description for the release